client/app/components/form/ConfirmSelectedOperatorForm.tsx

```diff
@@ -1,59 +1,141 @@
+  const [hasConfirmedOperator, setHasConfirmedOperator] = useState(false);
 
-  return (
-    <Form
-      schema={{}}
-      validator={validator}
-      onSubmit={async () => {
-        const response = await actionHandler(
-          "registration/select-operator/request-admin-access",
-          "POST",
-          `/dashboard/select-operator/user-operator/${operator_id}`,
-          {
-            body: JSON.stringify({
-              operator_id,
-            }),
-          },
-        );
-        if (response.error) {
-          setErrorList([{ message: response.error }]);
-          return;
-        }
+  const operatorHasAdminJSX: JSX.Element = (
+    <>
+      <p>
+        Looks like you do not have access to <b>{operator.legal_name}</b>
+        <br />
+        You will need the administrator of <b>{operator.legal_name}</b> to grant
+        you the access.
+      </p>
+      <p>Please confirm below if you would like to request access.</p>
+      <RequestAccessButton operatorId={operator.id} />
+    </>
+  );
 
-        push(
-          `/dashboard/select-operator/user-operator/${response.user_operator_id}`,
-        );
-      }}
-      className="flex flex-col w-64 mx-auto gap-2"
-    >
-      {errorList.length > 0 &&
-        errorList.map((e: any) => (
-          <Alert key={e.message} severity="error">
-            {e.message}
-          </Alert>
-        ))}
-      <SubmitButton
-        label="Request Access as Administrator"
-        classNames="w-max"
-      />
-    </Form>
+  const operatorHasNoAdmin: JSX.Element = (
+    <>
+      <p>
+        Looks like the operator <b>{operator.legal_name}</b> does not have an
+        administrator.
+      </p>
+      <p>Would you like to request access as its administrator?</p>
+      <RequestAccessButton operatorId={operator.id} isAdminRequest={true} />
+    </>
+  );
+
+  return (
+    <section className="text-center my-auto text-2xl flex flex-col gap-3">
+      {hasConfirmedOperator ? (
+        <>
+          <span>
+            <WarningIcon sx={{ color: "#ff0e0e", fontSize: 50 }} />
+          </span>
+          <div>{hasAdmin ? operatorHasAdminJSX : operatorHasNoAdmin}</div>
+          <Link
+            href=""
+            className="underline hover:no-underline"
+            style={{ color: BC_GOV_LINKS_COLOR }}
+            onClick={() => setHasConfirmedOperator(false)}
+          >
+            Go Back
+          </Link>
+        </>
+      ) : (
+        <>
+          <p>Kindly confirm if this is the operator that you represent.</p>
+          <Box display="flex" justifyContent="center">
+            <TableContainer
+              component={Paper}
+              sx={{ backgroundColor: DARK_GREY_BG_COLOR, width: "auto" }}
+            >
+              <Table sx={{ width: "auto" }}>
+                <TableBody>
+                  <TableRow>
+                    <TableCell sx={{ borderBottom: "none" }}>
+                      <b>Legal Name</b>
```

Comment:
legal name search doesn't seem to work? 
![image](https://github.com/bcgov/cas-registration/assets/77306764/fe7904a1-eead-4f7f-b283-9138112187ba)

Reply:
Since we want to show a single Operator on the next page, the search value should be identical to the operator's legal name.